server/application/rest/work_shift.py

```diff
@@ -125,7 +133,23 @@ def work_shifts():
         )
 
 def get_user_from_token(headers):
-    return headers["Authorization"]
+    token = headers.get("Authorization")
+    if not token:
+        return None, "No token provided"
+
+    try:
+        response = requests.get(
+            "https://api2-qa.gethelp.com/v1/users/current",
+            headers={"Authorization": f"Bearer {token}"}
+        )
+        response.raise_for_status()
+        return response.json(), None
```

Comment:
One problem I see is that what's being returned is the entire JSON object associated with the user. For example:
```
({'id': 6194, 'firstName': 'Kate', 'lastName': 'Holdener', 'dob': None, 'address1': None, 'address2': None, 'city': None, 'country': None, 'phone': '1234567890', 'receiveNewsletters': False, 'institutionIds': [], 'profileImageUrl': None, 'locationUpdateFrequency': 300, 'email': 'kate.holdener@slu.edu', 'state': None, 'zipCode': None}, None)

I think we'd want to identify the user by either email or their unique id. @AbhilashKotha @loganwyas what are your thoughts on this?
```

Reply:
May be we should try with id which is unique probably what are your thoughts @kate-holdener @AbhilashKotha @loganwyas

Reply:
Unique Id is a better option I guess